packages/shopify-codemod/transforms/assert/assert-to-assert-ok.js

```diff
@@ -0,0 +1,27 @@
+export default function assertToAssertOk({source}, {jscodeshift: j}, {printOptions = {}}) {
+  return j(source)
+    .find(j.CallExpression, {
+      callee: {
+        type: j.Identifier.name,
+        name: 'assert',
+      },
+      arguments: {
+        0: {
+          type: (type) => [
+            j.BinaryExpression.name,
+            j.CallExpression.name,
+            j.Identifier.name,
+            j.MemberExpression.name,
+            j.UnaryExpression.name,
```

Comment:
Fantastic question.  Short answer: it's trying to work around an issue that I should fix with another transform.

Long answer: It's  (verbosely) excluding `false` to avoid dodgy assertions like `assert(false, "something bad happened");` being affected by subsequent transforms.  Of course, there should be a `assert-false-to-assert-fail` transform 👍